flink/Dockerfile

```diff
@@ -1,21 +1,67 @@
+FROM openjdk:11.0.16-slim-buster
```

Comment:
Since we need only of JRE in runtime, we can save some megas of download here
```suggestion
FROM openjdk:11.0.16-jre-slim-bullseye
```

Reply:
Oh, my mistake, it should've been JRE-only from the start.